MainScreenStackWidget.py

```diff
@@ -0,0 +1,44 @@
+import sys
+from PySide6.QtWidgets import (
+    QApplication,
+    QMainWindow,
+    QVBoxLayout,
+    QPushButton,
+    QStackedWidget,
+    QLabel,
+)
+
+from StartMenuTest import (
+    StartMenu,
+)  # Assuming StartMenuTest.py contains StartMenu class
```

Comment:
interesting way to import this?
```suggestion
from StartMenuTest import StartMenu
```